helm-charts/support/values.jsonnet

```diff
@@ -46,10 +46,12 @@ function(VARS_2I2C_AWS_ACCOUNT_ID=null)
     alert: 'Server Startup Failed',
     expr: |||
       # We trigger any time there is a server startup failure, for any reason.
-      # The 'min' is to reduce the labels being passed to only the necessary ones
-      min(
-        jupyterhub_server_spawn_duration_seconds_count{status="failure"} > 0
-      ) by (namespace)
+      # The 'max' is to reduce the labels being passed to only the necessary ones
+      max by (namespace) (
+          (jupyterhub_server_spawn_duration_seconds_count{status="failure"} > 0)
+        -
+          ((jupyterhub_server_spawn_duration_seconds_count{status="failure"} offset 2m) > 0)
+      ) > 0
```

Comment:
Trigger an alert if any server has a new spawn failure (since 2m ago). 

Use max because $N(T)-N(T-2m)$ is positive for failures.